model/Notification/Response.ts

```diff
@@ -0,0 +1,27 @@
+import { isly } from "isly"
+import { Format } from "./Format"
+import { Tag } from "./Tag"
+import { Type } from "./Type"
+
+export interface Response {
+	id: string
+	title: string
+	body: string
+	format: Format
+	tag?: Tag
+	type: Type
+	unread?: boolean
```

Comment:
not optional

Reply:
Right, did not see an `@NotNull` - but I guess this is primitive type, so it can't be null.
```java
@Schema(description = "Has the current user read this notification?")
boolean unread;
```

Reply:
Removed